node/ledger/assembler_ledger.go

```diff
@@ -77,14 +84,14 @@ func NewAssemblerLedger(logger types.Logger, ledgerPath string) (*AssemblerLedge
 	}
 	logger.Infof("Assembler ledger opened block store: path: %s, ledger-ID: %s", ledgerPath, channelName)
 	ledger := fileledger.NewFileLedger(armaLedger)
-	transactionCount := uint64(0)
+	alm := AssemblerLedgerMetrics{}
 	height := ledger.Height()
 	if height > 0 {
 		block, err := ledger.RetrieveBlockByNumber(height - 1)
 		if err != nil {
 			return nil, fmt.Errorf("error while fetching last block from ledger %w", err)
 		}
-		_, _, transactionCount, err = AssemblerBatchIdOrderingInfoAndTxCountFromBlock(block)
+		_, _, alm.TransactionCount, err = AssemblerBatchIdOrderingInfoAndTxCountFromBlock(block)
```

Comment:
What about init of the number of blocks (height, its easy) and total size?
I don't mind that the total size will be estimated from the file sizes, or that we agree that it is the cumulative size from the beginning of the current process. Just document it.